vendor/k8s.io/kubernetes/staging/src/k8s.io/kubectl/pkg/generated/bindata.go

```diff
@@ -49,32 +51,21 @@ type bindataFileInfo struct {
 	modTime time.Time
 }
 
-// Name return file name
 func (fi bindataFileInfo) Name() string {
 	return fi.name
 }
-
-// Size return file size
 func (fi bindataFileInfo) Size() int64 {
```

Comment:
do we have another go-bindata than upstream?

Reply:
I didn't change the vendoring for go-bindata. It hasn't been updated since 2018 and I figured there was probably a good reason for that. Do you want to see a switch to the version upstream is using?

Reply:
if this change goes away, why not

Reply:
Given that upstream uses go-bindata/go-bindata and we're currently using jteeuwen/go-bindata in library-go and build-machinery-go that are also vendored, I'm assuming a switch involves updating those 2 repos to use the same module path before updating origin: 

https://github.com/openshift/build-machinery-go/pull/13
https://github.com/openshift/library-go/pull/766

I'm not sure of the wisdom of tying the rebase to these changes, though, so I'll wait for your confirmation before switching the rebase PR. This change could also be accomplished post-rebase.